README.md

```diff
@@ -460,7 +460,7 @@ avalanche-network-runner server \
 curl -X POST -k http://localhost:8081/v1/ping -d ''
 ```
 
-To start the cluster with custom VMs:
+To start the cluster with custom chains:
```

Comment:
I don't think `custom VMs` means the same thing as `custom chains`. I think `custom VM` means a non-subnet-evm or subnet-evm fork VM. I'm not sure if we want this to mean all vm types or just the non-subnet-evm ones

Reply:
We might want to just call this `chain` and not `custom chain`.

Reply:
ok, in order to be uniform, maybe I need to make such changes (just chain, not custom chain) on:
- some logs, vars, function names in local/blockchain.go, server/network.go, server/server.go
-  rpc related: custom_chains_healthy, custom_chains, CustomChainInfo

Do you agree? Do you also agree @holisticode  ?

Reply:
Well to me it sounds odd to just say "To start the cluster with chains". It's nuanced, because, yes, we could say a `custom VM` is a VM not provided by us, so a subnet-evm would not be a custom VM. However, a chain running with the subnet-evm is still a custom chain, as it is not a chain from our primary chains. In fact, if the user provides custom precompiles, a subnet-evm ceases to be vanilla and becomes a customized chain indeed...

Reply:
will keep as it is, based on private communications